cpp/ps0001/ps0001_soln.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+  // prompting user for 3 numbers to add 1.5/2
+  unsigned short num1, num2, num3;
+  cout << "please enter first number \n";
+  cin >> num1;
+  cout << "please enter second number \n";
+  cin >> num2;
+  cout << "please enter thired number \n";
+  cin >> num3;
+
+  // outputting the result of the summation process 3/3
+  cout << "the result is\t" << num1 + num2 + num3 << "\n";
+  
+  return 0;
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code